File: spinwords/spinwords.js
function spinSingleWord(str){
  return str.split('').reverse().join('');
}

function spinWords(str) {
var strArray = str.split(' ');
  for (var i = 0; i < strArray.length; i++) {
    if (strArray[i].length >= 5) {
      strArray[i] = spinSingleWord(strArray[i]);
    }
  }
  return strArray.join(' ');
}

File: spinwords/spinwords.rb
def spinWords(string)
  string.split.map{|i| i.size >= 5 ? i.reverse : i}.join " "
end

# Test.assert_equals(spinWords("Hey fellow warriors"), "Hey wollef sroirraw")
# Test Passed: Value == "Hey wollef sroirraw"